### DISCUSSION TITLE
error: while trying to get a Visual Studio environment, vcvarsall.bat returned 255

### DISCUSSION THREAD

**Discussion by D-zh-E** [microsoft/vcpkg#33298] [Q&A]
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
error: while trying to get a Visual Studio environment, vcvarsall.bat returned 255

AS before, when i try to: vcpkg install boost:x86-windows
It occur as before.

**Comment by bubble5144** [comment]
I also encountered this problem.

Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
error: while trying to get a Visual Studio environment, vcvarsall.bat returned 255

**Comment by autoantwort** [comment]
Maybe see https://github.com/microsoft/vcpkg/issues/17730#issuecomment-835149331
